src/scenes/Event/scenes/topic.js

```diff
@@ -1,4 +1,4 @@
-/* eslint react/no-array-index-key: 0 */ // Disable "Do not use Array index in keys" for options since they dont have unique identifier
+/* eslint react/no-array-index-key: 0, no-nested-ternary: 0 */ // Disable "Do not use Array index in keys" for options since they dont have unique identifier
```

Comment:
nit: is disabling this rule necessary? or more for convenience? suggest we try to follow the preset rules as much as possible.